sql/walrus--0.1.sql

```diff
@@ -112,7 +56,20 @@ Validates that the user defined filters for a subscription:
 - values are coercable to the correct column type
 */
 declare
-    col_names text[] = cdc.selectable_columns(new.entity);
+    col_names text[] = coalesce(
+            array_agg(c.column_name order by c.ordinal_position),
+            '{}'::text[]
+        )
+        from information_schema.columns c
+        where
+            (c.table_schema || '.' || c.table_name)::regclass = new.entity
```

Comment:
Should this be quoted with `quote_ident`?

Reply:
Yes it should, thanks!